packages/design-system/src/molecules/sidebar/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { useBreakpointValue } from 'native-base'
+import { PropsWithChildren } from 'react'
+import { Platform, ScrollView, View } from 'react-native'
+import { useThemeV2 } from '../../hooks'
+import { container } from '../../styles'
+import { DesktopSidebar } from './desktop'
+import { MobileSidebar } from './mobile'
+import { SidebarProps } from './types'
+
+export const Sidebar: React.FC<SidebarProps & PropsWithChildren> = ({
+  children,
+  ...props
+}) => {
+  const isMobile = useBreakpointValue({ base: true, xl: false })
+  const theme = useThemeV2()
+  const isNative = Platform.OS !== 'web'
+  if (isMobile) {
+    return (
+      <View>
+        <MobileSidebar {...props} />
+        {isNative ? <ScrollView>{children}</ScrollView> : children}
```

Comment:
We can avoid `isNative` variable declaration by using `Platform.select`.

```suggestion
        {Platform.select({ native: <ScrollView>{children}</ScrollView>, default: children })}
```